commons/src/main/java/io/hyscale/commons/logger/LoggerTags.java

```diff
@@ -18,7 +18,7 @@
 public enum LoggerTags {
 
 	USER_INFO_TAG("[INFO]"), ERROR("[ERROR]"), WARN("[WARN]"), VERBOSE("[VERBOSE]"), DEBUG("[DEBUG]"),
-	ACTION("[ACTION]"),;
+	ACTION("[ACTION]"), INPUT("[INPUT]");
```

Comment:
Can you justify for "Input" being a tag.

Reply:
Using INPUT tag when we are expecting user to pass an input, we can also use ACTION tag, but this seems more descriptive